src/gen_rules.ml

```diff
@@ -436,22 +442,32 @@ module Gen(P : Params) = struct
              [String.capitalize_ascii name]))
     in
     SC.add_rule sctx
-      (libs_and_cm >>>
+      (libs_and_cm
+       &&&
+       Build.fanout
```

Comment:
It allowed fewer underscores later on... if we were going to flatten it completely, we'd really need `Build.fanout4` - is that worth adding? (cf. a few lines later at `Build.run`)